darksidemanas.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "darksidemanas"
+project: "medical sos emergeny recordig and monitoring system"
+---
+
+# Project Name
+MEDICAL SOS EMERGENCY MONITORING AND RECORDING SYSTEM 
+## Summary
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB 
```

Comment:
```suggestion
1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTOM DESIGNED PCB 
```

Reply:
do you plan on following any guide of sorts for designing your pcb such that your pi can read analog signals?

Reply:
yup we can use the mcb3008 module to help pi to read analogue data

Reply:
do you already have access to that?

Reply:
yup if you want I can share bills and pictures

Reply:
no need hehe, thanks!